edit.html

```diff
@@ -33,10 +34,15 @@
 
     <script src="vendor/codemirror/addon/edit/matchbrackets.js"></script>
 
-    <link rel="stylesheet" href="vendor/codemirror/addon/lint/lint.css" />
+    <!-- CSSLint -->
```

Comment:
The stylesheet is part of CodeMirror's linting tooltips. I would think that should be loaded no matter which linter is selected. Also, since CSSLint is the default and small, should that be loaded by default? Or do you want it to be dynamically loaded?